script/utils/ScriptExtended.s.sol

```diff
@@ -177,7 +177,7 @@ abstract contract ScriptExtended is Script {
         return getAddressesJson(jsonFile, block.chainid);
     }
 
-    function getChainIdFromAddressessDirPath(string memory path) internal pure returns (uint256) {
+    function getChainIdFromAddressesDirPath(string memory path) internal pure returns (uint256) {
```

Comment:
this function might be used somewhere hence before I merge it all the references must be updated too